www/modules/centreon-open-tickets/providers/Abstract/AbstractProvider.class.php

```diff
@@ -1391,14 +1393,14 @@ protected function saveHistory($db_storage, &$result, $extra_args = array())
                         `service_id`,
                         `service_state`,
                         `service_description`
-                    ) VALUES ('" .
-                        $db_storage->escape($result['ticket_id']) . "', '" .
-                        $db_storage->escape($row['host_id']) . "', '" .
-                        $db_storage->escape($row['host_state']) . "', '" .
-                        $db_storage->escape($row['host_name']) . "', '" .
-                        $db_storage->escape($row['service_id']) . "', '" .
-                        $db_storage->escape($row['service_state']) . "', '" .
-                        $db_storage->escape($row['description']) . "'
+                    ) VALUES (
+                        '" . $db_storage->escape($result['ticket_id']) . "',
+                        '" . $db_storage->escape($row['host_id']) . "',
```

Comment:
breakline after or before the concatenate operator rather than in string

Reply:
@loiclau I do not understand here, the idea is to better organize the view of the query. Do you want to add a breakline to each concatenation?

Reply:
yes if you have a long string query, but if you have already concatenate operator break the line on it.

Reply:
the problem is the confusion that is in the query string separation, look how is the end of line: ```
                    "INSERT INTO mod_open_tickets_data (
                        `ticket_id`, `subject`, `data_type`, `data`
                    ) VALUES ('" .
                        $db_storage->escape($result['ticket_id']) . "', '" .
                        $db_storage->escape($extra_args['subject']) . "', '" .
                        $db_storage->escape($extra_args['data_type']) . "', '" .
                        $db_storage->escape($extra_args['data']) . "'
                    )"
```
This broke the structure of the query. Neither form breaks the style guide. however I found it confusing as was the structure of the query. If you can exemplify how I would like it to be, please show me. ;-)